src/_guides/libraries/objective-c-interop.md

```diff
@@ -83,6 +83,10 @@ The `output` is the path of the Dart file that ffigen will create.
 The entry point is the header file containing the API.
 In this example, it is the internal `AVAudioPlayer.h` header.
 
+If you are creating an iOS plugin, you may need to reference 
```

Comment:
In the above config file we show a header file for a specific library on MacOS. But many developers will use this for iOS plugins. I had trouble finding the header files for commonly used frameworks on iOS, so I thought it would be helpful to point in that direction. We could certainly include it as an aside or a note though, since its a different use case then the example

Reply:
Sounds like a great idea for a note then :)

Perhaps a tip?: `{{site.alert.tip}}`